src/features/profile/index.tsx

```diff
@@ -79,19 +80,21 @@ const Profile: React.SFC = () => (
         monthlyCost,
         certificateUrl,
       } = insurance;
-      const { name: cashbackName, imageUrl: cashbackImageUrl } = cashback;
 
       return (
         <Container>
           <Header>
             <Spacing height={8} />
-            <CashbackImage
-              source={{ uri: cashbackImageUrl }}
-              resizeMode="contain"
-            />
+            {cashback && (
```

Comment:
```suggestion
            {cashback ? (
                          <CashbackImage
                source={{ uri: cashback.imageUrl }}
                resizeMode="contain"
              />
            ): <SelectCashback />}
```

Suggestion may be a bit off but you get the idea.